crates/frontend/src/schema.rs

```diff
@@ -81,14 +81,18 @@ pub enum SchemaType {
 impl SchemaType {
     pub fn default_value(&self) -> Value {
         match self {
-            SchemaType::Multiple(_) => json!(""),
-            SchemaType::Single(JsonSchemaType::String) => json!(""),
+            SchemaType::Multiple(_) => Value::String(String::default()),
```

Comment:
Can we write a comment on how to use SchemaType::Mulitple with some documentation at the type location?

Reply:
@ShubhranshuSanjeev this definitely needs documentation. I assumed this represented a way to define multiple schemas for a type